catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -109,7 +107,6 @@ public AdminPollerServiceBean(ConfigurationAdmin configurationAdmin, org.osgi.se
 
     public void init() {
         serviceFactoryFilter = getServiceFactoryFilterProperties();
-        serviceFilter = getServiceFilterProperties();
```

Comment:
It technically isn't being used but it is a protected method. Possibly something else in the package could use it at some time. What do you think? If I get rid of that method would I need to change anything else (like documentation)?